ivadomed/scripts/download_data.py

```diff
@@ -210,6 +247,8 @@ def install_data(url, dest_folder, keep=False):
     shutil.rmtree(extraction_folder)
 
 
+install_data.__doc__%=create_string(DICT_URL)
```

Comment:
i would definitely add comments above this line of code to explain what it does

Reply:
done in 100747d